src/test/java/com/virtuslab/archunit/MethodCallsTestSuite.java

```diff
@@ -0,0 +1,58 @@
+package com.virtuslab.archunit;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.constructors;
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.methods;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import com.tngtech.archunit.core.domain.JavaCodeUnit;
+import com.tngtech.archunit.core.domain.JavaMethodCall;
+import com.tngtech.archunit.lang.ArchCondition;
+import com.tngtech.archunit.lang.ConditionEvents;
+import com.tngtech.archunit.lang.SimpleConditionEvent;
+import com.tngtech.archunit.lang.syntax.elements.GivenCodeUnits;
+import org.junit.Test;
+
+public class MethodCallsTestSuite extends BaseArchUnitTestSuite {
+
+  @Test
+  public void methods_calling_messageBusConnection_subscribe_must_call_Disposer_register_later_too() {
+    String subscribeMethodCallString = "com.intellij.util.messages.MessageBusConnection.subscribe(com.intellij.util.messages.Topic, java.lang.Object)";
+    String registerMethodCallString = "com.intellij.openapi.util.Disposer.register(com.intellij.openapi.Disposable, com.intellij.openapi.Disposable)";
+
+    List<GivenCodeUnits<? extends JavaCodeUnit>> codeUnits = Arrays.asList(constructors(), methods());
+    for (GivenCodeUnits<? extends JavaCodeUnit> c : codeUnits) {
+
+      c.should(new ArchCondition<JavaCodeUnit>(
+          "calling ${subscribeMethodCallString} must call ${registerMethodCallString} too") {
```

Comment:
```suggestion
          "call ${registerMethodCallString} if they call ${subscribeMethodCallString}") {
```

So that the error reads better (currently it's like `methods should calling ... must call ... too' was violated`)